m0t1fs/linux_kernel/file.c

```diff
@@ -3090,13 +3091,15 @@ static int pargrp_iomap_dgmode_recover(struct pargrp_iomap *map)
 		rc = -EIO;
 		goto end;
 	}
+#if !ISAL_ENCODE_ENABLED
```

Comment:
We should not check the "ISAL_ENCODE_ENABLED" here.
We should do this checking in m0_parity_recov_mat_gen():
if ISAL_ENCODE_ENABLED is not defined, m0_parity_recov_mat_gen() will do the same.
If ISAL_ENCODE_ENABLED is define, m0_parity_recov_mat_gen() should be empty.
But the checking should happen in m0_parity_recov_mat_gen() function in sns/parity_math.c file, not here.

Reply:
Done.